### PR TITLE
Added keyboard accessibility to tomato slider

### PR DESCRIPTION
**Pull request by LJMNZS**
Ctr+LeftArrow and Ctr+RightArrow will now increase and decrease the number of tomatoes in the slider when entering a new task.

### CODE REVIEW

**Comment by xingyuzzzhu** [comment]
Doesn't seem to work on mac

**Comment by LJMNZS** [comment]
> Doesn't seem to work on mac

Did you try with the command key or the control key? Or did it not work with either

File: source/js/components/TomatoSlider.js [hunk 1]
```diff
@@ -110,6 +116,38 @@ class TomatoSlider extends HTMLElement {
...
+   * @param {KeyboardEvent} e
+   */
+  handleKeyDown(e) {
+    this.keysPressed[e.key] = true;
+    if (this.keysPressed.Control && this.keysPressed.ArrowLeft && !e.repeat) {
+      this.input.value = Math.max(this.input.value - 1, 1);
+      this.colorTomatos(this.input.value, 'red');
+    }
+    if (this.keysPressed.Control && this.keysPressed.ArrowRight && !e.repeat) {
+      if (this.input.value === '1') {
```
Comment (HarrisSteven) [inline source/js/components/TomatoSlider.js:130]: Can we replace these with `this.input.value = Math.min(this.input.value + 1, 5);`?

Comment (LJMNZS) [reply source/js/components/TomatoSlider.js:130]: We tried that, but this.input.value is a string, so it concatenates 1 to the value which breaks the functionality. It works for the max value on the left key with subtract, but we weren't sure why.

Comment (HarrisSteven) [reply source/js/components/TomatoSlider.js:130]: We could cast it with `Number(...)`

Comment (markbussard) [reply source/js/components/TomatoSlider.js:130]: Have you guys looked into something like https://www.w3schools.com/jsref/jsref_parseint.asp to get a number?

**Pull request closed without merging**